config/utils.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func getConfigYaml(path string) ([]byte, error) {
+	if strings.HasPrefix(path, "http") {
+		res, err := http.Get(path)
+		if err != nil {
+			return []byte{}, err
+		}
+
```

Comment:
If there's no error doing the `Get` call, we can add a defer function to close the response body
```suggestion
defer res.Body.Close()
```